stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/list/MessageContainer.kt

```diff
@@ -86,12 +88,14 @@ public fun MessageContainer(
             onQuotedMessageClick = onQuotedMessageClick,
         )
     },
+    typingIndicatorContent: @Composable (TypingItemState) -> Unit = { }
```

Comment:
Yes, we have the model for the xml sdk for the typing indicator item to be inside the list. This will expose the same functionality for the compose sdk.